pandas/core/strings.py

```diff
@@ -1967,6 +1997,10 @@ def _validate(data):
         values = getattr(data, "values", data)  # Series / Index
         values = getattr(values, "categories", values)  # categorical / normal
 
+        # explicitly allow TextDtype
+        if isinstance(values.dtype, TextDtype):
+            return "string"
```

Comment:
so this is the inference path?

Reply:
I'm not sure when / where `_validate` is called, but this is check is done before any inference is done. We're only operating on the dtype.

I suspect that if / when we add a BytesDtype / ByteArray EA and disallow `.str` on object-dtype this method can be removed entirely.